tasks/test.js

```diff
@@ -1,7 +1,31 @@
 import _, { keys } from 'lodash';
+import fs from 'fs';
+import path from 'path';
+
 const visualRegression = require('../utilities/visual_regression');
 
 module.exports = function (grunt) {
+  grunt.registerTask('checkPlugins', 'Checks for plugins which may disrupt tests', function checkPlugins() {
+    const done = this.async();
+    const pluginsDir = path.resolve('./plugins/');
```

Comment:
I was hoping there was a better way to get this path as opposed to assembling it, but that doesn't appear to be the case.